### PR TITLE
Docs release 1.8.0 csi-3528 general updates

### PR DESCRIPTION
**Pull request by rivkap**
Please approve this PR based off of CSI-3528 (1.8.0 - general formatting updates), including:

-  CSI-3402 prefix book_files dir with a dot 
-  CSI-3403 align md filenames plurality

### CODE REVIEW

File: docs/content/configuration/creating_volumereplicationclass.md [hunk 1]
```diff
@@ -6,9 +6,9 @@ Create a VolumeReplicationClass YAML file to enable volume replication.
 
 In order to enable volume replication for your storage system, create a VolumeReplicationClass YAML file, similar to the following `demo-volumereplicationclass.yaml`.
 
-When configuring the file, be sure to use the same array secret and array secret namespace as defined in [Creating a Secret](csi_ug_config_create_secret.md).
+When configuring the file, be sure to use the same array secret and array secret namespace as defined in [Creating a Secret](creating_secret.md).
 
-For information on obtaining your storage system `system_id`, see [Finding a `system_id`](csi_ug_config_replication_find_systemid.md).
+For information on obtaining your storage system `system_id`, see [Finding a `system_id`](finding_systemid.md).
```
Comment (oriyarde) [inline docs/content/configuration/creating_volumereplicationclass.md:11]: @rivkap it is also important to mention that the `system_id` should be of the `VolumeReplication`s' `replicationHandle`

Comment (rivkap) [reply docs/content/configuration/creating_volumereplicationclass.md:11]: In volumereplicationclass I don't see specifically a replicationHandle.


```
apiVersion: replication.storage.openshift.io/v1alpha1
kind: VolumeReplicationClass
metadata:
  name: demo-volumereplicationclass
spec:
  provisioner: block.csi.ibm.com
  parameters:
    system_id: demo-system-id
    copy_type: async  # Optional. Values sync/async. The default is sync.

    replication.storage.openshift.io/replication-secret-name: demo-secret
    replication.storage.openshift.io/replication-secret-namespace: default
```

Should I add this text (in **bold**)?

**Make sure that the `system_id` is added as part of the `replicationHandle`.** For information on obtaining your storage system `system_id`, see [Finding a `system_id`](finding_systemid.md).

Comment (oriyarde) [reply docs/content/configuration/creating_volumereplicationclass.md:11]: first, let me rephrase myself with more details, so that we are clear about the missing info.
as already implied by the documentation, multiple `VolumeReplication`s can be created per `VolumeReplicationClass`.
the missing info is:
the `system_id` in a `VolumeReplicationClass` should be the id of the storage system which contains the volumes that "match" (i.e. having their id as) the `replicationHandle` in the `VolumeReplication`s that will be created from aforementioned `VolumeReplicationClass`. hope it's clear.

second, how can we simplify this phrasing?

Comment (rivkap) [reply docs/content/configuration/creating_volumereplicationclass.md:11]: @oriyarde What about - 

Use the same `system_id` as the storage system containing the volumes as the `VolumeReplication` `replicationHandle` used to create this `VolumeReplicationClass`.

Comment (oriyarde) [reply docs/content/configuration/creating_volumereplicationclass.md:11]: 1. a `VolumeReplication` is not used to create a `VolumeReplicationClass`, it is the other way around.

2.
>same system_id as the storage system

>the volumes as the `VolumeReplication` `replicationHandle`

I'm not sure an id can be "as" the object that has it.

Comment (rivkap) [reply docs/content/configuration/creating_volumereplicationclass.md:11]: updated - 
Use the system_id of the storage system containing the volumes as the VolumeReplication replicationHandle used for creating this VolumeReplicationClass.

Comment (oriyarde) [reply docs/content/configuration/creating_volumereplicationclass.md:11]: > the VolumeReplication replicationHandle used for creating this VolumeReplicationClass

this sounds like `VolumeReplication`s are used to create `VolumeReplicationClass`s.
but, as I [wrote in 1.](https://github.com/IBM/ibm-block-csi-driver/pull/403#discussion_r815815980), that is never the true for "classes". "classes" are used to create non-classes; a "snapshot class" is used to create "snapshots", a "replication class" is used to create "replications", etc...

Comment (rivkap) [reply docs/content/configuration/creating_volumereplicationclass.md:11]: Use the system_id of the storage system containing the `replicationHandle` volumes.
